packages/web/src/components/feature-flag-override-modal/FeatureFlagOverrideModal.tsx

```diff
@@ -39,8 +39,8 @@ const setOverrideSetting = (flag: string, val: OverrideSetting) => {
 
 export const FeatureFlagOverrideModal = () => {
   const hotkeyToggle = useDevModeHotkey(70 /* f */)
+  const hotkeyRef = useRef<boolean | null>(null)
```

Comment:
maybe a comment explaining how this is used. generally this reads a bit weird to me, but it works fine. not worth further stewing over since it's dev UI